clients/acm.d.ts

```diff
@@ -0,0 +1,350 @@
+import {Request} from '../lib/request';
+import {Response} from '../lib/response';
+import {AWSError} from '../lib/error';
+import {Service} from '../lib/service';
+declare class ACM extends Service {
+  /**
```

Comment:
That's an interesting idea. I don't think this will work for S3, or services that have custom config options (anything with dualstack, just S3 for now) if we stop exporting the service interfaces though. I'll take a look.